src/main/kotlin/racing/domain/Car.kt

```diff
@@ -1,16 +1,22 @@
-package racing
+package racing.domain
+
+import java.util.Collections
 
 class Car(val name: String, private val probability: Probability = Probability()) {
     var position = START_POSITION
         private set
-    val records = ArrayList<Int>()
+    private val records = mutableListOf<Int>()
 
     fun tryGo() {
         if (probability.canGo())
             position++
         records.add(position)
     }
 
+    fun getRecords(): List<Int> {
+        return Collections.unmodifiableList(records)
```

Comment:
[toList()](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/to-list.html)를 이용해보는 건 어떨까요?
```suggestion
        return records.toList()
```